wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -723,6 +725,49 @@ func ScreenOpenCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (s
 	return update, nil
 }
 
+func ScreenReorderCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sstore.UpdatePacket, error) {
+    // Resolve the UI IDs for the session and screen
+    ids, err := resolveUiIds(ctx, pk, R_Session|R_Screen)
+    if err != nil {
+        return nil, err
+    }
+
+    // Extract the screen ID and the new index from the packet
+    screenId := ids.ScreenId
+    newScreenIdxStr := pk.Kwargs["index"]
+    if newScreenIdxStr == "" {
+        return nil, fmt.Errorf("missing new screen index")
+    }
+    newScreenIdx, err := strconv.Atoi(newScreenIdxStr)
```

Comment:
can use resolvePosInt(newScreenIdxStr, 1).  the default case won't get used because of your check above.  this will also protect against 0 and negative numbers.